bundles/org.openhab.core/src/main/java/org/openhab/core/internal/i18n/LanguageResourceBundleManager.java

```diff
@@ -167,11 +170,11 @@ public String getText(String resource, String key, Locale locale) {
      *
      * @return the translated text, or null if the key could not be translated
      */
-    public String getText(String key, Locale locale) {
+    public @Nullable String getText(@Nullable String key, @Nullable Locale locale) {
         return getText(null, key, locale);
     }
 
-    private String getTranslatedText(String resourceName, String key, Locale locale) {
+    private @Nullable String getTranslatedText(String resourceName, String key, Locale locale) {
```

Comment:
The parameters may be `Nullable` as well. According to the [docs](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/util/ResourceBundle.html#getBundle(java.lang.String,java.util.Locale,java.lang.ClassLoader,java.util.ResourceBundle.Control)), it would cause a NullpointerException in `getBundle`, which leads to returning null.

Not sure if this is the way to go, but at least it would be more consistent with the other methods above.